<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBook</class>
 <widget class="QDialog" name="addBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加图书</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#addBook{border-image: url(:/pic/background_blurred.jpg);}
QLineEdit {
	border: 2px solid rgba(180, 180,180,0.3);
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: rgba(204, 204, 204,0.2); /* 背景颜色 */
	color: rgb(50,50,50); /* 文本颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	/*font-size: 20pt; /* 文本字体大小 */
	letter-spacing: 1px;
}

QLineEdit:focus{ /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid rgba(160, 160,160,0.8);
	border-radius: 6px;
	background-color:  rgba(180, 180, 180,0.2);
	color: rgb(40,40,40);
	selection-background-color: rgba(142,210,255,0.6); /* 选中文本的背景颜色 */
	selection-color: rgb(40,40,40);
}

QPushButton {background-color: rgba(160,160, 160,0.2);padding: 5px 6px 5px 6px;font: 27px 'Microsoft YaHei';letter-spacing: 1px;border: transparent;border-radius: 6px;}
QPushButton:pressed:hover {background-color: rgba(153, 153, 153,0.5);}
QPushButton:hover {background-color: rgba(230, 230, 230,0.4);border: 2px solid rgba(180,180,180,0.8)}
QPushButton:disabled {background-color: rgba(204, 204, 204,0.3);color: rgb(120, 120, 120);}
QLabel{letter-spacing: 1px;}
QPushButton{letter-spacing: 2px;}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="isbn_Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>13</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入13位ISBN码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="search_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(245, 245, 245,0.8);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="attitude_label">
   <property name="geometry">
    <rect>
     <x>742</x>
     <y>16</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>82</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>86</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>141</y>
     <width>78</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>140</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>200</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="press_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>198</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>258</y>
     <width>120</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版日期：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pressDate_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>256</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>316</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版地：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pressPlace_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>314</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>372</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>372</y>
     <width>241</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>433</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>页数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>488</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pages_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>430</y>
     <width>271</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="words_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>488</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>83</y>
     <width>72</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分类：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="clc_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>221</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>376</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_Btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tip">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 0, 0, 0.6);</string>
   </property>
   <property name="text">
    <string>手动输入时，请认真核对ISBN码无误后，再点击查询!切勿多次快速连续查询，可能会封IP!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="bookDesc_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	border: 2px solid rgba(180, 180,180,0.3);
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: rgba(204, 204, 204,0.2); /* 背景颜色 */
	color: rgb(50,50,50); /* 文本颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	/*font-size: 20pt; /* 文本字体大小 */
	letter-spacing: 1px;
}

QTextEdit:focus{ /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid rgba(160, 160,160,0.8);
	border-radius: 6px;
	background-color:  rgba(180, 180, 180,0.2);
	color: rgb(40,40,40);
	selection-background-color: rgba(142,210,255,0.6); /* 选中文本的背景颜色 */
	selection-color: rgb(40,40,40);
}

QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 8px;
}
 
QScrollBar::handle:vertical
{
      background-color: rgba(150,150,150,1);
      min-height: 30px;
	  border-radius:3px;
	  width: 6px;
}
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{	
		background:transparent;
      	border-radius:3px;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:14pt; font-weight:400; font-style:normal; letter-spacing:1px;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>图书描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shelfNum_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#shelfNum_Edit{
	border: 2px solid rgb(255, 170, 0);
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: rgba(204, 204, 204,0.2); /* 背景颜色 */
	color: rgb(40,40,40); /* 文本颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	/*font-size: 20pt; /* 文本字体大小 */
	letter-spacing: 0px;
}

#shelfNum_Edit:disabled{ /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid rgba(214, 217,219,1);
	border-radius: 8px;
	background-color:  rgba(228, 233, 236,0.5);
	color: rgb(40,40,40)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>84</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书架号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>isbn_Edit</zorder>
  <zorder>search_Btn</zorder>
  <zorder>attitude_label</zorder>
  <zorder>name_Edit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>author_Edit</zorder>
  <zorder>label_3</zorder>
  <zorder>press_Edit</zorder>
  <zorder>label_4</zorder>
  <zorder>pressDate_Edit</zorder>
  <zorder>label_5</zorder>
  <zorder>pressPlace_Edit</zorder>
  <zorder>label_6</zorder>
  <zorder>price_Edit</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>pages_Edit</zorder>
  <zorder>words_Edit</zorder>
  <zorder>label_9</zorder>
  <zorder>clc_Edit</zorder>
  <zorder>label_10</zorder>
  <zorder>quit_Btn</zorder>
  <zorder>Tip</zorder>
  <zorder>bookDesc_Edit</zorder>
  <zorder>addBtn</zorder>
  <zorder>shelfNum_Edit</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
